examples/ensemble/plot_gradient_boosting_regression.py

```diff
@@ -61,9 +110,11 @@
 plt.xlabel('Boosting Iterations')
 plt.ylabel('Deviance')
 
-# #############################################################################
+##############################################################################
 # Plot impurity-based feature importance
+# -------------------------------------
 #
+# Next we will plot the impurity-based feature importance.
```

Comment:
This sentence does not bring any new info that is not already present in the title of the section. Let's remove it for conciseness.